operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -137,7 +131,8 @@ func (r *reconcilePVCExtensionRun) Execute() error {
 	if r.centralObj.DeletionTimestamp != nil || r.persistence == nil {
 		return r.handleDelete()
 	}
-	if common.ObsoletePVC(r.centralObj.GetAnnotations()) {
+	// Starting from 4.6, we do not create or own PVCTargetCentral
+	if r.target == PVCTargetCentral {
 		return r.handleDelete()
```

Comment:
This actually seems to be unreachable code, since for `target == PVCTargetCentral` the `getPersistenceByTarget` always returns `nil`, which causes the `handleDelete` on line 138 (132 in changed file) to run. Perhaps consider folding this into the `if` above? 🤔